### PR TITLE
Analysis Completed. Review Requested.

### PR DESCRIPTION
**Pull request by gyutaepark**
Analysis is completed and stored in analysis folder.

### CODE REVIEW

File: cyclus_input/predicting_the_past.xml [hunk 1]
```diff
@@ -21,6 +22,7 @@
       <Sink>
         <in_commods>
           <val>casked_spent_uox</val>
+          <val>tails</val>
```
Comment (jbae11) [inline cyclus_input/predicting_the_past.xml]: `caksed_spent_uox` is HLW,
while `tails` is LLW.
The prototype `Sink` is deployed in timestep 960, which is half accurate,
considering there are already LLW (or tails) repositories around.
I suggest creating a separate `sink` for LLW which is present from the beginning of the simulation.

File: cyclus_input/buildtimes/deployinst.xml [hunk 2]
```diff
@@ -113,6 +114,7 @@
         <val>Comanche_Peak_2</val>
         <val>Watts_Bar_1</val></prototypes>
     <build_times>
+        <val>900</val>
```
Comment (jbae11) [inline cyclus_input/buildtimes/deployinst.xml]: `caksed_spent_uox` is HLW,
while `tails` is LLW.
The prototype `Sink` is deployed in timestep 960, which is half accurate,
considering there are already LLW (or tails) repositories around.
I suggest creating a separate `sink` for LLW which is present from the beginning of the simulation.

File: cyclus_input/predicting_the_past.xml [hunk 3]
```diff
@@ -71,6 +75,7 @@
         <feed_recipe>uf6</feed_recipe>
         <product_commod>enr_u</product_commod>
         <tails_commod>tails</tails_commod>
+        <tails_assay>0.001</tails_assay>
```
Comment (jbae11) [inline cyclus_input/predicting_the_past.xml]: Double check me on this one, but maybe 0.002 is a more reasonable number?

File: cyclus_input/predicting_the_past.xml (hunk 1 above)
Comment (gyutaepark) [inline cyclus_input/predicting_the_past.xml]: Thanks, I'll make the changes

File: cyclus_input/predicting_the_past.xml (hunk 3 above)
Comment (gyutaepark) [inline cyclus_input/predicting_the_past.xml]: Yup, that's also fixed.
Thanks

File: cyclus_input/buildtimes/deployinst.xml (hunk 2 above)
Comment (gyutaepark) [inline cyclus_input/buildtimes/deployinst.xml]: Fixed.

**Comment by gyutaepark** [comment]
@jbae11 
I've made the power plot. Unfortunately, it does not match or even come close to the actual published values.

**Pull request merged**